website/docs/core-concepts/formats.md

```diff
@@ -4,4 +4,23 @@ sidebar_position: 5
 
 # Formats
 
-## Coming Soon!
\ No newline at end of file
+Formats for a space can be created from the Formats menu. Default formats for Dega can be added by clicking on the Create default Formats. 
+
+The default formats for Dega are
```

Comment:
The *current* default formats available in Dega are:

Reply:
updated